docker/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM java:openjdk-8-jre-alpine
```

Comment:
How to verify this? There seems no error reported when I start 3 bookies, and did some simple write.